tbx/static_src/sass/components/_animated-icon.scss

```diff
@@ -0,0 +1,334 @@
+@use 'config' as *;
+
+.animated-icon {
+    $root: &;
+
+    &--megaphone {
+        transform: rotate(0);
+        display: inline-flex;
+        align-items: center;
+        gap: 3px;
+        transition: transform $transition;
+
+        @include reduced-motion() {
+            transition: none;
+        }
+
+        #{$root}__megaphone {
+            width: 35px;
+            height: 30px;
+
+            @include media-query(large) {
+                width: 55px;
+                height: 50px;
+            }
+        }
+
+        #{$root}__noise {
+            width: 15px;
+            height: 25px;
+            transition: transform $transition;
+            transform: translate3d(0, 0, 0);
+
+            @include media-query(large) {
+                width: 20px;
+                height: 45px;
+            }
+
+            @include reduced-motion() {
+                transition: none;
+            }
+        }
+
+        a:focus &,
+        a:hover & {
```

Comment:
All the animations are based on them having an `<a>` tag as a parent but we could include buttons as well if we deem it necessary.